app/assets/stylesheets/layout/helper-classes.scss

```diff
@@ -82,6 +82,12 @@
       text-align: right !important;
     }
   }
+
+  &--medium-up {
+    @include breakpoint(medium) {
+      text-align: right !important;
```

Comment:
yeah, the idea is that all of the helper classes should have really high specificity because they are usually meant to override other classes. 

For example, let's say I had an element in DE that was like:
.subtotal { background: white; text-align: left; }
 
but maybe in this one instance I wanted to align the text to the right, so I would need the !important to override the original style

Most of the classes in this file have them, the ones that don't are mostly old and just haven't been updated.